synapse/lib/storm.py

```diff
@@ -859,11 +859,9 @@ def cmpr(tufo):
         return cmpr
 
     def _cmprCtorRange(self, oper):
-
-        prop = self._reqOperArg(oper, 'prop')
```

Comment:
@vEpiphyte @therealsilence - it didn't seem like there was a way to actually get this to work.

Example storm query:
`intform +range(intform, (-1,0))` (lifts all intform, then filters out intforms less than -1 or greater than 0)

We need to make this (or a similar) change if want the filter to work like the example above.

Reply:
If the above change is OK, I can make it in the `seen` filter as well

Reply:
I'm fine with this change and we can fix the rest of the filter operators in a second pass